src/__tests__/sample-pdf.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { createWriteStream, unlink } from 'fs';
+
+import request from 'supertest';
+
+import extractPDFText from '../../scripts/text-extraction';
+import app from '../app';
+import { getTotalPages } from '../pdf';
+import { generateTmpPath } from '../util/fileSystem';
+import testPayloads from './payloads.json';
+
+beforeAll(done => {
+  setTimeout(done, 5000);
```

Comment:
Why is the timeout needed?

Reply:
It's for warmup, like letting puppeteer launch and other startup scripts to run.